docs/products/kafka/kafka-connect/howto/request-new-connector.rst

```diff
@@ -0,0 +1,14 @@
+Request a new connector
+=======================
+
+If you know about new and interesting Apache Kafka®ive connectors you'd like us to support, please send us an email to support@Aiven.io about it to help us shaping the future roadmap. We will evaluate the requested connector and might add support for it.
```

Comment:
This "interesting Apache Kafka®ive connectors" should be "interesting Apache Kafka® connectors"?